src/Resources/Resources/ChangeLog.md

```diff
@@ -18,7 +18,9 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
-* Fix for null reference bug in GetAzureRoleAssignmentCommand
+* Fixed for null reference bug in `Get-AzRoleAssignment`
```

Comment:
Is this `Fixed four null reference bugs` or `Fixed a null reference bug`?

Reply:
This message comes from another people's commit. I just changed its tense using past tense.